drizztsaga/drizztsaga.tp2

```diff
@@ -58,6 +58,7 @@ ALWAYS
 	INCLUDE ~DRIZZTSAGA\lib\worldmap_update.tph~
 	INCLUDE ~DRIZZTSAGA\lib\del_are_trigger.tph~
 	INCLUDE ~DRIZZTSAGA\tpa\ds-core.tpa~
+	INCLUDE ~drizztsaga\tpa\items_creatures.tpa~
```

Comment:
I would name the file "nerfing.tpa", because items and creatures are still copied in the ds-core

Reply:
done